trino/trino.go

```diff
@@ -136,6 +135,8 @@ const (
 	kerberosRealmConfig      = "KerberosRealm"
 	kerberosConfigPathConfig = "KerberosConfigPath"
 	SSLCertPathConfig        = "SSLCertPath"
+
+	CHAN_SIZE = 10
```

Comment:
We might want to try a different value here, maybe 2 is sufficient.

Reply:
How big a single response can be?

Reply:
It's 1MB by default: https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/server/protocol/ExecutingStatementResource.java#L80

Reply:
Then I guess 10 is fine. Unless it does not improve anything compared to 2. Then we can go lower. I guess it will depend on number of cores we have and if we are actually able to read data quickly enough to saturate CPU - i kinda doubt that :)

Reply:
The benchmark returns the same results when using unbuffered channels, so I removed this. The benchmark doesn't do any data processing, so no actual application can be faster than that and won't benefit from additional buffering. Storing both a single HTTP response and the decoded payload is enough buffering.